taichi/python/export_ggui.cpp

```diff
@@ -140,11 +140,20 @@ struct PyScene {
              FieldInfo indices,
              bool has_per_vertex_color,
              py::tuple color_,
-             float width) {
+             float width,
+             float draw_index_count,
+             float draw_first_index,
+             float draw_vertex_count,
+             float draw_first_vertex) {
     RenderableInfo renderable_info;
     renderable_info.vbo = vbo;
     renderable_info.indices = indices;
     renderable_info.has_per_vertex_color = has_per_vertex_color;
+    renderable_info.has_user_customized_draw = true;
+    renderable_info.draw_index_count = (int)draw_index_count;
```

Comment:
OOC: why are these passed in as floats while they're mainly used as ints?

Reply:
Test may be diffcult to design, I will consider that in followup PR.

Here, I think user may usually use divide caculations to make a part of renderables visible, so they may forget to convert float to int type in python scope, so I convert that to int type in cpp part. In addition, the display of an index is often not that important, and the user can still control the value if they want. But if you want these args' data type to be consistent, it's also okey.
For example, there are 100 verticles which represent some particles, and the user want the middle part(1/3 ~ 2/3) to be visible. They should use like this:
```python
draw_vertex_count = int(100/3)
draw_vertex_offset = int(100/3)
```
vs.
```python
draw_vertex_count = 100/3
draw_vertex_offset = 100/3
```
Under the second situation, I think user usually do not care about if 1 particle is visible or not.